test/features/step_definitions/pupil-spa/completePageSteps.js

```diff
@@ -0,0 +1,29 @@
+// Step definitions
+/* global browser expect */
+
+const {defineSupportCode} = require('cucumber')
+defineSupportCode(function ({Given, When, Then}) {
+  Given(/^I am on the complete page$/, function () {
+    return browser.get(this.config.spaUrl + 'check/complete')
```

Comment:
This is fine for now, but just a heads-up that there is a chance that this URL will be included inside the check in the future.